webcompat/templates/dashboard/triage/stats.html

```diff
@@ -33,7 +33,29 @@
         -ms-flex-pack: center;
             justify-content: center;
   }
-  
+
+  .wc-Stat--alert {
+    background-color: #c12600;
+    animation: pulse 5s infinite;
+  }
+
+  @keyframes pulse {
```

Comment:
I thought it was cool to have the slight animation too… then… when testing CPU went 30% higher.
@magsout do you experience the same thing?

Reply:
not 30%, but 15%. although on home of webcompat, only 0.8%, hum weird, it's only a small infinite animation.. So I remove it